src/node_v8.cc

```diff
@@ -60,6 +60,14 @@ void GetHeapStatistics(const FunctionCallbackInfo<Value>& args) {
 
 
 void SetFlagsFromString(const FunctionCallbackInfo<Value>& args) {
+  Isolate* isolate = args.GetIsolate();
+  Environment* env = Environment::GetCurrent(isolate);
+
+  if (args.Length() != 1)
+    return env->ThrowTypeError("v8 flag is required");
```

Comment:
Maybe make that `args.Length() < 1`?  Or simply drop it and rely on the `args[0]->IsString()` check below to catch it.